connectivity/connectivity-demos/src/com/axonivy/connectivity/rest/log/filter/RequestLoggingFilter.java

```diff
@@ -0,0 +1,18 @@
+package com.axonivy.connectivity.rest.log.filter;
+
+import java.io.IOException;
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.container.ContainerRequestFilter;
+import javax.ws.rs.ext.Provider;
+import ch.ivyteam.log.Logger;
+
+@Provider
```

Comment:
I think this filter could be always active, due to the @provider annotation.
if you omit it, it will only work for the rest-clients where the feature is registered?

Reply:
Yes, you were right, once I left it out, it only worked when I added it as a feature 👍.